pkg/storaged/details.js

```diff
@@ -34,7 +34,25 @@ define([
      */
 
     function init_details(client, jobs) {
-        var type, name;
+        var type, name, state;
+
+        function ensure_state_multipathd_running() {
+            if (state.multipathd_running === undefined) {
```

Comment:
what happens if this is set to `false` (deemed not running) and I then start the service manually in the console?